web-api/src/ingestion/routes.rs

```diff
@@ -224,7 +224,7 @@ impl UnvalidatedDocumentForIngestion {
             )
                 .into())
             }
-            (true, false) => PreprocessingStep::CuttersSplit,
+            (true, false) => PreprocessingStep::NltkSplitV1,
```

Comment:
Maybe we can have `PreprocessingStep::default_split()` to keep the information of which the default is near the preprocessing module.